programs/fileio.h

```diff
@@ -42,44 +42,52 @@ extern "C" {
 ***************************************/
 typedef enum { FIO_zstdCompression, FIO_gzipCompression, FIO_xzCompression, FIO_lzmaCompression, FIO_lz4Compression } FIO_compressionType_t;
 
+typedef struct ZSTDIO_prefs_s ZSTDIO_prefs_t;
+
+ZSTDIO_prefs_t* ZSTDIO_defaultPreferences(void);
```

Comment:
some minor details, on naming convention : 

1. All symbols in this unit are prefixed `FIO_`.
Is there a reason to prefer prefixing these new symbols with `ZSTDIO_` instead ?

2. `ZSTDIO_defaultPreferences()` is likely inspired by `LZ4IO_defaultPreferences()`, so that's understandable.
I would nonetheless prefer `*_createPreferences()` instead if possible.

Reasoning : I would expect `*_defaultPreferences()` to return the structure itself, by value.
But what's happening here is slightly different : we get a pointer to some heap memory area.
As a consequence, the caller must also remember to deallocate this space afterwards.

`create` helps to realize there is this responsibility at stake.

Reply:
Fixed both naming issues.